networking_ccloud/tests/unit/common/test_config_validation.py

```diff
@@ -33,10 +33,12 @@ def test_validate_ml2_vlan_ranges_success(self):
         cfg.CONF.set_override('network_vlan_ranges', ['seagull:23:42', 'cat:53:1337'], group='ml2_type_vlan')
         cfg.CONF.set_override('driver_config_path', 'invalid/path/to/conf.yaml', group='ml2_cc_fabric')
         validate_ml2_vlan_ranges(self.conf_drv)
+        cfg.CONF.clear_override('tenant_network_types', group='ml2')
```

Comment:
Didn't you want to remove the `set_override()` instead? If not, this needs to be done differently, because the `clear_override()` needs to happen if any of the previous steps fail, too. It's currently written for a working test only.